daemon/uevent.c

```diff
@@ -707,6 +711,53 @@ handle_udev_event(struct uevent *uevent, char *raw_p)
 	IF_TRUE_RETURN_TRACE(strncmp(uevent->subsystem, "usb", 3) ||
 			     strncmp(uevent->devtype, "usb_device", 10));
 
+	uint16_t short_id_length = 14;
+	//TODO: move reference id to global config file?
+	//TODO: replace check short_id with ID.contain("VendorString/TokenName")
+	const char token_short_id[] = "0000000000013E";
+		
+	if (!strncmp(uevent->id_serial_short,token_short_id,short_id_length))
```

Comment:
don't hard code ID => check on each inserted USB device if it is used as a container token